### DISCUSSION TITLE
Configure Font features

### DISCUSSION THREAD

**Discussion by niekdomi** [zed-industries/zed#21385] [Feature Requests]
I want to use font features of the *FiraCode*-Font

unfortunately I can't figure out based on the docs how to set them for zed, although it should be described here: https://zed.dev/docs/configuring-zed#buffer-font-features

Here's the link to FiraCode: https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets

Here's my config:

```json
  "buffer_font_family": "FiraCode Nerd Font",
  "buffer_font_features": {
    "cv01": true,
    "cv02": true,
    "cv15": true,
    "cv19": true,
    "cv23": true,
    "cv29": true,
    "cv31": true,
    "ss01": true,
    "ss03": true,
    "ss05": true
  },
```

**Comment by martinffx** [comment]
```
  "ui_font_family": "Fira Code",
  "buffer_font_family": "Fira Code"
```

this worked for me, as long as the font is installed by that name on my system there is not issue.

**Comment by niekdomi** [comment]
this enables the font but doesn't set the font features

**Reply by niekdomi** [reply]
But as far as I know, this feature isn't implemented yet or it's an issue with `cosmic-text`

**Comment by MBestKing** [comment]
Hi, i guess im sooo late but i tried it but with 
```json
  "buffer_font_family": "FiraCode Nerd Font",
  "buffer_font_features": {
     "liga": true, // Add this
     "calt": true, // also this too
     "cv01": true,
     "cv02": true,
     "cv15": true,
     "cv19": true,
     "cv23": true,
     "cv29": true,
     "cv31": true,
     "ss01": true,
     "ss03": true,
     "ss05": true
  }
```
And it works, so you needed to enable: "liga": true, "calt": true
for it to work with font signuares.
